data/blog/kafka-monitoring-opentelemetry.mdx

```diff
@@ -204,9 +204,9 @@ This use case is designed to cater scenarios where platform owners want to get a
 
 **Approach**
 
-In this proposal, we plan to have two tabs listing producer and consumer services grouped by topics. A user would be able to see details about producer and consumer services, by clicking on a particular topic and service group, and accessing a side panel displaying more detailed RED (Rate, Error, Duration) metrics for each partition.
+In this proposal, we plan to have two tabs listing producer and consumer services grouped by topics. A user would be able to see details about producer and consumer services, and on clicking a particular Topic & Service group, they can see a side panel that will show more detailed RED (Rate, Error, Duration) metrics for each partition.
 
-Powered by traces, users can dive deeper into partitions where p99 latency or error rates are high. By clicking on the latency/error rate values, they will be taken to detailed trace list pages showing the spans with high latency or errors. Users can then further investigate the traces to hypothesize the underlying issues.
+Since, all of these are powered, users can dive deeper into partitions where p99 latency or error rates are high. By clicking on the latency/error rate values, they will be taken to detailed trace list pages showing the spans with high latency or errors. Users can then further investigate the traces to hypothesize the underlying issues.
```

Comment:
all of these are powered ->  all of these are powered by traces

Reply:
my bad